docs/source/additional/release.rst

```diff
@@ -362,6 +362,111 @@ These sets of versions have been tested for compatibility:
 Changelog
 ---------
 
+Substra 0.21.0 (first OS release) - 2022-09-12
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This is our first open source release since 2021! The product was renamed Substra.
```

Comment:
The product was already named Substra for public users, Connect was a closed-source name

Reply:
@RomainGoussault

Reply:
```suggestion
This is our first open source release since 2021! When the product was closed source it used to be named Connect. It is now renamed Substra.
```

Reply:
I don't get why we say to the user that it used to be named Connect, but as you prefer